### ISSUE TITLE
Briefing text box overflows screen width

### ISSUE THREAD

**Issue by Sniperhid** [TMF3/TMF#147]
Screenshot courtesey of Aliensteak.
![](https://cdn.discordapp.com/attachments/98696477882204160/335885989501206528/20170716021802_1.jpg)

Suggestion: use the `max` command and calculate the biggest width based off the end of the screen and subtract a small margin. Essentially `((screen width - x) - margin)` should be the largest width value.

**Comment by headswe** [comment]
```
_menuPos = ctrlPosition _menu;
_xPos = (_menuPos select 0) + (_menuPos select 2) + _margin;
_width = safeZoneW - _xPos - _margin;
```

**Comment by Sniperhid** [comment]
Fixed in #153
